src/tests/tests/storage.js

```diff
@@ -76,11 +76,28 @@
       {dnt: false, heuristicAction: "", nextUpdateTime: 100, userAction: ""});
     assert.ok(action_map.getItem('testsite.com').nextUpdateTime === 100);
 
-    let newValue = {dnt: false, heuristicAction: "", nextUpdateTime: 101, userAction: ""};
+    let newValue = {dnt: true, heuristicAction: constants.BLOCK, nextUpdateTime: 99, userAction: constants.USER_BLOCK};
     action_map.merge({'testsite.com': newValue});
 
-    // Merged in object should have new nextUpdateTime value
+    // userAction should be merged
+    assert.ok(action_map.getItem('testsite.com').userAction === constants.USER_BLOCK);
```

Comment:
Unit test nit: should use `assert.equal` instead as it's more to the point; we are comparing two values, not just checking if something is true.

Reply:
We should include a human-friendly assertion message to explain what it is we are testing, especially when there are many assertions in a single test. I guess the comments here could become the messages.